apps/prairielearn/src/pages/instructorAssessmentStatistics/instructorAssessmentStatistics.ts

```diff
@@ -31,33 +37,52 @@ router.get(
     await updateAssessmentStatistics(res.locals.assessment.id);
 
     // re-fetch assessment to get updated statistics
-    const assessmentResult = await sqldb.queryOneRowAsync(sql.select_assessment, {
-      assessment_id: res.locals.assessment.id,
-    });
-    res.locals.assessment = assessmentResult.rows[0].assessment;
+    res.locals.assessment = await sqldb.queryRow(
+      sql.select_assessment,
+      {
+        assessment_id: res.locals.assessment.id,
+      },
```

Comment:
Nit:
```suggestion
      { assessment_id: res.locals.assessment.id },
```

Reply:
Updated in [fe614f8](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/fe614f896046789604dac2796569dc2b7421cebc).